mob.go

```diff
@@ -1063,9 +1098,49 @@ func start(configuration Configuration) error {
 	sayInfo("you are on wip branch '" + currentWipBranch.String() + "' (base branch '" + currentBaseBranch.String() + "')")
 	sayLastCommitsList(currentBaseBranch.String(), currentWipBranch.String())
 
+	if configuration.OpenLastModifiedFile{
+		openLastModifiedFileIfPresent(configuration)
+	}
+
 	return nil // no error
 }
 
+func openLastModifiedFileIfPresent(configuration Configuration) {
+	debugInfo("OpenLastModifiedFile1")
```

Comment:
These debugInfos I would remove before release.

Reply:
I find these kind of checks scary. It means that we're handling this on a very low level, but everywhere. The invariant here is that it will always be either an empty string, or a specific string, but never anything in between. I would want to isolate this invariant in its own type. So that we abstract the low level detail away from the rest of the code. But that's just how I see it.

Reply:
done.